### ISSUE TITLE
enable RDP option for macOS systems running xrdp

### ISSUE THREAD

**Issue by tlcarpenter** [Ylianst/MeshCentral#2017]
I compiled and installed xrdp

http://xrdp.org/

following this guide

https://ryancreecy.com/2019/10/29/xrdp-on-mac.html

on a macOS Big Sur system and was able to successfully connect to the system using RDP over SSH with Remmina. I'd like to test using RDP from MC2 to this system: is there an easy way to enable the RDP option on an MC2 server for connections to macOS systems?

**Comment by Ylianst** [comment]
Right now, MeshCentral Router is only on Windows, so there is no super easy way to make RDP work over MeshCentral. You do have the option to download "MeshCMD" for macOS, that binary can be launched from the terminal and allows you to map a local port on your mac to a remote port on any MeshCentral device. Once you get the hang of it, it's pretty easy... but not your usual macOS pretty.

**Comment by tlcarpenter** [comment]
My apologies, I assumed MC2's RDP capability was implemented entirely by the MC2 server; found the statement "The router link will only show up on Windows browsers." in the MC2 Router  UG.

Unfortunately the current version of xrdp won't solve the issues that exist when using noVNC since xrdp just proxies an RDP connection to the macOS VNC server anyway.

Prior to experimenting with xrdp I had tried using meshcmd but didn't get that to work. I'll give that another try and open a new issue if need be. Thanks for the feedback.
